index.html

```diff
@@ -1,41 +1,47 @@
-        Вы можете найти гайды по верстке в интернете и шаг за шагом учиться превращать PSD-макеты в веб-страницы.
-        На этом этапе вы уже можете пробоваться на должность HTML-верстальщик.
-        <h3><em>JavaScript</em></h3>
-        Это язык программирования, который позволяет вам создать динамически обновляемый контент, управляет мультимедиа, анимирует изображения.
-
-      <h2>Советы:</h2>
-    <p>
-        1 Копайте глубже <br>
-      Каждый браузер по-разному отображает один и тот же код, но для эффективной работы нужно, чтобы ваш код был кроссбраузерным и адекватно отображался для всех пользователей. Для этого потратьте немного больше времени и разберитесь, в каком из браузеров код работает правильно и поправьте его, для адекватного отображения в остальных. Таким образом, вы сделаете правильные выводы и сэкономите время в будущем. Это касается и ситуаций, если вы слепо добавляете что-то в код до тех пор, пока не получите нужный результат. Разберитесь, чтобы не наступать на грабли каждый раз.
-      </p>
-      <p>
-          2 Читайте код <br>
-          Читая код, написанный другими разработчиками, вы улучшаете свои навыки, видите, что у задач бывает не только один вид решения. Также вы можете принять участие в открытых проектах, код которых дополняют и правят разные разработчики. Это улучшает ваши навыки, добавляет способности критически смотреть на код.
-      </p>
-
-      <p> 3 Работайте в команде <br>
-      Работа на фрилансе - это мечта всех разработчиков, однако работая в команде, вы имеете отличную возможность перенимать опыт старших девелоперов, учиться, советоваться и совершенствоваться.
-      </p>
-<p>
-    <a href="about/index.html">Об авторе</a>
-</p>
+    <div class="container">
+      <div class="top-section">
+        <div class="block-1">
+          <div class="header block-1">
+            <div class="header logo-img"></div>
+            <div class="header menu">
+              <p class="header item-1">Stays</p>
+              <p class="header item-1">Attractions</p>
+              <div class="header item-2"></div>
+              <div class="header item-3"></div>
+            </div>
+          </div>
+        </div>
+        <div class="line"></div>
+        <div class="page">
+          <div class="page-discover">
+            <h1 class="title-discover">
+              Discover stays <br />
+              to live, work or just relax
+            </h1>
+          </div>
+          <div class="page-names">
+            <div class="names">Your destination or hotel name</div>
+            <div class="names">Check-in — Check-out</div>
+          </div>
+          <form class="form-search">
+            <p class="form-search-info">
+              <input type="text" class="form-city" placeholder="New York" /><input type="text" class="form-date" placeholder="Tue 15 Sept — Sat 19 Sept" /><input type="text" class="form-room" placeholder="2 Adults — 0 Children — 1 Room" /><button class="form-submit">Search</button>
```

Comment:
The code style here leaves much to be desired.

Reply:
Ok, what do you mean? Can you give some details?

Reply:
The row 36 is rather large:
1) it is difficult to read it coz it doesn't fit into the screen; 
2) comments to this row will be without reference.

Reply:
But if I split they won't be in one row. How can I do it in another way?

Reply:
```suggestion
              <input type="text" class="form-city" placeholder="New York" />
              <input type="text" class="form-date" placeholder="Tue 15 Sept — Sat 19 Sept" />
              <input type="text" class="form-room" placeholder="2 Adults — 0 Children — 1 Room" />
              <button class="form-submit">Search</button>
```

Reply:
Not sure what you mean, but try to follow the recommendations.

Reply:
I understood what you meant, but if did it the form wouldn't be as I wanted in a row without space, it would have space between inputs in design page.
I've got it that I have to use another way to do this search-row but how I don't know. Any tips?

Reply:
Look, it doesn't affect on appearance in the browser. You should repeat the lesson about DOM. Or just check your concept in the browser to understand that there are no restrictions which you mind.

Reply:
What lesson's number do you mean? 
I've checked this topic, it's from Javascript lesson 23
![image](https://user-images.githubusercontent.com/98038966/183252162-eb90e346-c1ed-41cb-a733-202a7b68e9a0.png)

Reply:
It is not available now